spec/javascripts/govuk_publishing_components/lib/govspeak/govspeak-track-links-spec.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env jasmine */
+var GOVUK = window.GOVUK
+
+describe('Govspeak Track Links', function () {
+  var tracker
+  var element
+
+  GOVUK.analytics = GOVUK.analytics || {}
+  GOVUK.analytics.trackEvent = function () {}
+
+  describe('Single link', function () {
+    beforeEach(function () {
+      spyOn(GOVUK.analytics, 'trackEvent')
+
+      element = document.createElement('div')
+      element.setAttribute('data-track-links-category', 'Content page 1')
+      element.innerHTML = '<a href="/blah/blahhhh">Blahh</a>'
+
+      tracker = new GOVUK.Modules.GovspeakTrackLinks()
+      tracker.start(element)
+    })
+
+    afterEach(function () {
+      GOVUK.analytics.trackEvent.calls.reset()
```

Comment:
Looks like there's a bit of repetition here. Can I suggest putting the `spyOn(GOVUK.analytics, 'trackEvent')` in a `beforeEach` for all the tests (i.e. put it after line 7, before the internal `describe` functions) and likewise the `GOVUK.analytics.trackEvent.calls.reset()` in an `afterEach` for all the tests.

I'd also recommend wrapping the `GOVUK.analytics.trackEvent.calls.reset()` in a `if (GOVUK.analytics.trackEvent.calls) {` as well.